target/linux/ath79/dts/qca9563_glinet_gl-x1200.dtsi

```diff
@@ -182,8 +174,8 @@
 		compatible = "qcom,ath10k";
 		reg = <0 0 0 0 0>;
 
-		nvmem-cells = <&macaddr_art_5006>, <&calibration_ath10k>;
-		nvmem-cell-names = "mac-address", "pre-calibration";
+		nvmem-cells = <&calibration_ath10k>;
+		nvmem-cell-names = "pre-calibration";
```

Comment:
Is the driver also looking in the `pre-calibration` attribute for mac addresses?

Reply:
https://github.com/openwrt/openwrt/commit/650d37a809bdb84a517ff0580bb2c4168e501d0d implies it is.